core/src/main/java/bisq/core/app/BisqSetup.java

```diff
@@ -692,6 +692,9 @@ private void initDomainServices() {
 
         accountAgeWitnessService.onAllServicesInitialized();
         signedWitnessService.onAllServicesInitialized();
+        if (user.getRegisteredArbitrator() != null) {
```

Comment:
Why not do it inside of signedWitnessService? User can be injected. Better to not spread out responsibilities...

Reply:
Yes, perhaps that's neater. Fixed